micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/SignUp-v2/CarouselComponent/CarouselComponent.js

```
@@ -0,0 +1,160 @@
+                height: '8px',
+                borderRadius: '50%',
+                border: 'none',
+                padding: 0,
+                background: index === currentIndex ? 'white' : 'rgba(255, 255, 255, 0.4)',
+                cursor: 'pointer',
+                transition: 'all 0.3s ease',
+                transform: index === currentIndex ? 'scale(1.4)' : 'scale(1)',
+                boxSizing: 'border-box',
+              }}
+            />
+          ))}
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)